File: src/main/webapp/js/service/sellproductService.js
//服务层
app.service('sellproductService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../sellproduct/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../sellproduct/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../sellproduct/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../sellproduct/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../sellproduct/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../sellproduct/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../sellproduct/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/top/axbt/to/service/SellproductService.java
package top.axbt.to.service;
import java.util.List;

import top.axbt.to.domain.TbSellproduct;
import top.axbt.to.entity.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface SellproductService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbSellproduct> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbSellproduct sellproduct);
	
	
	/**
	 * 修改
	 */
	public void update(TbSellproduct sellproduct);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbSellproduct findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbSellproduct sellproduct, int pageNum, int pageSize);

	public PageResult findPage(TbSellproduct sellproduct, int pageNum, int pageSize,int pro_level);
	
}
